statsmodels/tsa/statespace/initialization.py

```diff
@@ -0,0 +1,607 @@
+"""
```

Comment:
Is this just a a non-cython implementation of `_initialization.pyx`?

Reply:
No this is a wrapper. I'll explain more in the main comments.